chirp/drivers/kguv8e.py

```diff
@@ -552,6 +552,7 @@ def _get_dcs(val):
             return code, pol
 
         tpol = False
+        print(mem.number, _mem.txtone, _mem.rxtone)
```

Comment:
Leftover debug? Please remove :)

Reply:
Sorry for that. There seems to be a bug with this driver when trying to decode/encode DCS codes which results in some memories not being loaded at all. I tried to look into it but had no success!